controllers/bsl.go

```diff
@@ -196,9 +197,76 @@ func (r *DPAReconciler) ReconcileBackupStorageLocations(log logr.Logger) (bool,
 			)
 		}
 	}
+
+	_, err := r.syncBSLs(&dpa)
+
+	if err != nil {
+		return false, err
+	}
+
+	return true, nil
+}
+
+func (r *DPAReconciler) syncBSLs(dpa *oadpv1alpha1.DataProtectionApplication) (bool, error) {
+	bslLabels := map[string]string{
```

Comment:
dpaBslLabels?

labels for BSLs managed via dpa.

Reply:
Done